rasa/core/channels/telegram.py

```diff
@@ -33,11 +34,13 @@ async def send_text_message(
         self, recipient_id: Text, text: Text, **kwargs: Any
     ) -> None:
         for message_part in text.strip().split("\n\n"):
+            logger.debug(f"message_part: {message_part}")
             self.send_message(recipient_id, message_part)
 
     async def send_image_url(
         self, recipient_id: Text, image: Text, **kwargs: Any
     ) -> None:
+        logger.debug(f"image: {image}")
         self.send_photo(recipient_id, image)
 
     async def send_text_with_buttons(
```

Comment:
Re: line 51, it seems like vertical is the default now (using moodbot0? I would double check that passing these `button_type` args works as expected

Reply:
fixed